src/main/java/baseball/view/InputView.java

```diff
@@ -0,0 +1,40 @@
+package baseball.view;
+
+import baseball.constant.Constant;
+import camp.nextstep.edu.missionutils.Console;
+
+public class InputView {
+
```

Comment:
```suggestion
private InputView() {
}
```

스태틱 필드, 메서드로만 이뤄진 클래스는 불필요한 객체 생성을 막기위해 private 기본 생성자를 사용하는 것이 좋을 것 같습니다.😀

Reply:
아하 불필요한 객체 생성을 막기위해서라도 생성자를 개발자가 사용하지 않음에도 불구하고 private 접근 제어자를 설정하면 좋군요?

Reply:
말씀 주신 내용을 공부해봤어요.

명시적 private 생성자를 통해서 불필요한 인스턴스 생성을 방지하는거군요. 정말 감사게 배워갑니다!